### DISCUSSION TITLE
Making my own grand organ sample set from other sample sets

### DISCUSSION THREAD

**Discussion by chuckkamas** [GrandOrgue/grandorgue#1613] [Help]
I have an old Rodgers 660 organ that I am in the process of converting to MIDI. I have the manuals and pedals converted using my own PCBs (braging a bit here). The next step is to get the stops working. To that end I would like to "remix" sample sets. I.e. take some samples from one online organ and some from another to make a uber set that has all of the stops of the original Rodgers. Yep, it will be a stealth GO. However, I have not been able to figure out how to do this despite much googling. Can someone point me in the right direction?  Chuck

**Comment by larspalo** [comment]
1. Get all sample sets you want to use material from.
2. Get https://github.com/larspalo/GOODF
3. Learn to use GOODF
4. Create your dream organ

**Reply by Bumblebee001** [reply]
I suggest you'd use dry samples. Wet samples in different sets vary in "wetness" because they largely reflect the different buildings the real instruments are in - from small chapels with little reverb to huge cathedrals with several seconds of reverb. The amount of reverb in sample-sets also depends on how the various instruments were recording (mics close to pipes = dry, mics positioned further away capture more of the building's acoustics) and how they were processed. You will have a hotch-potch of acoustics if you use wet samples. Dry is dry and all samples will be more or less the same. You can then apply reverb (when **playing** the sample-set you have created) through third party software that will effect all samples evenly.

**Reply by Bumblebee001** [reply]
You can also apply reverb by adding an IR (Impulse Response) file in GrandOrgue itself.

![image](https://github.com/GrandOrgue/grandorgue/assets/6999990/51f9312d-852e-46d6-9dac-f08bce10bf5f)

The program complains that this feature is not yet supported but I tried it out before and it seemed to work.... so I am not sure why this window pops up!

**Comment by chuckkamas** [comment]
Lars

Thank you for all of your work making this tool. I will definitely give it
a try.
